app.py

```diff
@@ -20,9 +20,15 @@
 import random
 import logging
 from pottery import synchronize
-logging.basicConfig(level=logging.DEBUG)
-r = redis.from_url(os.environ.get("REDIS_URL"))
 
+import jwt#sai
+from jwt import PyJWKClient#sai- need to run "pip3 install -U pyjwt" because jwt and pyjwt modules conflict
+
+
+logging.basicConfig(level=logging.DEBUG)
+#print(os.environ.get("REDIS_URL"))
+#r = redis.from_url(os.environ.get("REDIS_URL")) or "redis://"
+r = "redis://"
```

Comment:
This needs to go back to the regular redis command.

Reply:
Ok. Not sure if this matters but I had issues using the original REDIS URL on my Ubuntu (via WSL2). A little digging told me that it has to do with my Ubuntu+REDIS version.